src/components/InputWithLabel.jsx

```diff
@@ -23,4 +24,9 @@ const InputWithLabel = (props) => {
   );
 };
 
+InputWithLabel.PropTypes = {
```

Comment:
Same thing here `InputWithLabel.propTypes` instead of `InputWithLabel.PropTypes`
Difference is the capital "P"